tests/run-multitests.py

```diff
@@ -92,20 +92,25 @@ def start_file(self, filename, prepend="", append=""):
 
 
 class PyInstanceSubProcess(PyInstance):
-    def __init__(self, cmd):
-        self.cmd = cmd
+    def __init__(self, argv, env=None):
+        self.argv = argv
```

Comment:
Just curious: what's the reason to rename `cmd` to `argv`?  Is it because it's a list not a single str?

Reply:
Yes exactly that. (`cmd` inplied a full command line to me, i.e. like you'd pass to `system()`)